test/suites/drift/suite_test.go

```diff
@@ -66,7 +66,7 @@ var _ = Describe("Drift", Label("AWS"), func() {
 		})
 		// choose an old static image
 		parameter, err := env.SSMAPI.GetParameter(&ssm.GetParameterInput{
```

Comment:
musing: Can we create a function that completely encapsulates the behavior of getting an older AMI version and gives the id back rather than leaking the SSM parameter details into the test context?

Reply:
Will implement in a follow up PR